src/Network/NWEthernetChannel.cs

```diff
@@ -24,10 +25,16 @@
 
 namespace Network {
 
+#if !NET
 	[NoWatch, NoTV, NoiOS, Mac (10,15)]
+#endif
```

Comment:
Missing mac10.15 for net core, looking at the min versions I have: [assembly: SupportedOSPlatform ("macos10.14")]

Reply:
it's fine, can't annotate `delegate` on dotnet

Reply:
that's lame